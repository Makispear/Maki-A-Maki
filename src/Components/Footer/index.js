import React from "react";

function Footer() {
  return (
    <footer>
      <h3 data-testid="copyright" className="text-tertiary">&copy; Maki A. Maki, 2021</h3>
    </footer>
  )
}

export default Footer
