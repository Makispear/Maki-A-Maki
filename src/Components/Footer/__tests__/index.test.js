import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup)

describe('Footer Component', () => {
  it('renders', () => {
    render(<Footer />)
  })

  it('has a copyright sign', () => {
    const { getByTestId } = render(<Footer />)

    expect(getByTestId('copyright')).toContainHTML('Â©');
  })

  it('Matches snapshot', () => {
    const { asFragment } = render(<Footer />)

    expect(asFragment).toMatchSnapshot()
  })
})
