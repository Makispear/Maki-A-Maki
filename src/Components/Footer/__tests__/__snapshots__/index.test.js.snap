// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Matches snapshot 1`] = `[Function]`;
